lib/cog/commands/helpers.ex

```diff
@@ -150,4 +152,20 @@ defmodule Cog.Commands.Helpers do
     end)
   end
 
+  defp changeset_errors(changeset) do
+    msg_map = Ecto.Changeset.traverse_errors(changeset,
+                                             fn
+                                               {msg, opts} ->
+                                                 Enum.reduce(opts, msg, fn {key, value}, acc ->
+                                                   String.replace(acc, "%{#{key}}", to_string(value))
+                                                 end)
+                                               msg ->
+                                                 msg
+                                             end)
+
+    msg_map
+    |> Enum.map(fn({field, msg}) -> "#{field} #{msg}" end)
+    |> Enum.join("\n")
+  end
```

Comment:
Really wish Phoenix came with a function that does this. Had to write something similar for Flywheel.